src/test/test-uid-range.c

```diff
@@ -20,6 +20,9 @@ TEST(uid_range) {
         assert_se(uid_range_covers(p, 0, 0));
         assert_se(!uid_range_covers(p, 0, 1));
         assert_se(!uid_range_covers(p, 100, UINT32_MAX));
+        assert_se(!uid_range_covers(p, -1, 1));
```

Comment:
uh, -1 is outside of the uid_t range. this is hence an implicit cast, which is ugly, even for a test case, and equivalent to UINT32_MAX. hence, please change this to UINT32_MAX